### ISSUE TITLE
Shell and Fin sprite is completely broken after update

### ISSUE THREAD

**Issue by Yowbaby** [Regalis11/Barotrauma#9352]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

**SHELL PROBLEM:** 

When changing the sprite scale of certain shells, the sprite seems to have an off-texture effect, this is a pretty serious bug because it affects the aesthetic of all ships in the game and can be a game-breaking factor for many players out there, especially sub-builder. Please consider this hotfix ASAP.

_Please refer to photos below:_
![image](https://user-images.githubusercontent.com/93289071/173902787-be5b757d-1b20-4d3b-b0b6-9a953aaeb7df.png)

![image](https://user-images.githubusercontent.com/93289071/173903440-14fc80d9-10af-46c5-8824-7d80ffe0c665.png)

**FIN PROBLEM:**

Fins also face the same issue, I don't think much is needed to be repeated: 

_''fin special part 2 bug''_
![image](https://user-images.githubusercontent.com/93289071/173904026-62486710-a4dd-417e-a2db-08aabd9dbd0a.png)

_''tail fin f1 bug''_
![image](https://user-images.githubusercontent.com/93289071/173904129-175f107c-c2a2-45ff-a8fc-2f940b55d06e.png)

**These are just some of the examples that I used as _reference_ and I'm fairly certain that the sprite issues may have a deeper root among other entities in the game. A deeper re-evaluation is required from the developer side. Thanks!**

### Bug prevalence

Happens every time I play

### Version

Other

### -

v0.18.11.0

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Seems to have been caused by the fix we did to address https://github.com/Regalis11/Barotrauma/issues/8922

Fixed in https://github.com/Regalis11/Barotrauma-development/commit/94324b014591376a60ad19a994163b856df9870d

**Comment by MaaiG** [comment]
![Image 001_17-04](https://user-images.githubusercontent.com/93348499/174088333-4a499491-fcbc-44fa-a508-640074246d34.png)

**Comment by Rokvach** [comment]
Looks like this fix caused issues on some vanilla subs, this is from Dugong
![image](https://user-images.githubusercontent.com/7299826/174233427-d438205a-a126-45e8-8449-c500906ecb56.png)

**Comment by Regalis11** [comment]
Decided to revert those fixes for now and reopen https://github.com/Regalis11/Barotrauma/issues/8922

**Comment by NilanthAnimosus** [comment]
Tested against release commit https://github.com/Regalis11/Barotrauma-development/commit/36e3a2755f5fc201a748a32b63cc93a8eb30d90a,

The above issues involving tiling do not seem to be present with the various fins and background pieces I have tried scaling and mirroring with, meanwhile a few of the problems found in issue #8922 (Such as shuttle fin B at scale 0.6) I can see are a problem again.

I do believe the original fix attempt that was reverted with a small tweak (a small mistake that was missed where one Y should be an X) should work and fix these problems based on my own tests with it.

My tests did reveal a few minor sprite issues unrelated to this and the other similar mentioned issue (Minor artifacts with some derelict sprites and one fin piece having a thin line of pixels I'm not sure its meant to) but I don't think anyone would notice those (and are not tied to tiling/mirroring/scaling issues in those cases but perhaps just more sprite cleanup).
